shared/cfg/guest-os/Linux/Fedora/31.cfg

```diff
@@ -0,0 +1,107 @@
+        # Unattended-file does not require any changes
+        unattended_file = unattended/Fedora-31.ks
+        unattended_install.url:
+            # Installation works fine with mem=1024 on methods such as cdrom
+            # but fails ("No space left on device") with methods such as url.
+            mem = 2048
+            url = http://dl.fedoraproject.org/pub/fedora-secondary/releases/31/Server/${vm_arch_name}/os/
+        # ARCH dependent things
+        aarch64:
+            arm64-mmio:
+                # Only latest updates contain virtio driver
+                inactivity_watcher = none
+                take_regular_screendumps = no
+            kernel_params = "console=ttyAMA0 console=ttyS0 serial"
+            unattended_install.cdrom:
+                md5sum_cd1 = 5629a2ae883f8afbc5f675f94798398c
+                md5sum_1m_cd1 = eebbca4eb53a276d79063fe30ddc6558
+            unattended_install.url:
+                url = http://dl.fedoraproject.org/pub/fedora/linux/releases/31/Server/${vm_arch_name}/os
+                sha1sum_vmlinuz = f1bc444f795b5c3eb7f7cc229bfa5abb
+                sha1sum_initrd = 63c069ce4a4057abc23dcd54897fb03e
+        ppc64:
+            kernel_params = "console=hvc0 serial"
+            boot_path = ppc/ppc64
+            unattended_install.cdrom:
+                md5sum_cd1 = fdbc46e96cd1c210ab62f7a3ad51b938
+                md5sum_1m_cd1 = 2ef20babadfa30544fd9e94f9d3b573f
+            unattended_install.url:
+                sha1sum_vmlinuz = a682caf6ea9718cf48bda511f464a8a0
+                sha1sum_initrd = 824ae6ec75a39117b873ea85c48a5038
+        ppc64le:
+            kernel_params = "console=hvc0 serial"
+            boot_path = ppc/ppc64
+            unattended_install.cdrom:
+                md5sum_cd1 = c01d0d04b481c56380c88ea2c6e11ce5
+                md5sum_1m_cd1 = 0c78f0233155031b8f4183fb78631ffe
+            unattended_install.url:
+                sha1sum_vmlinuz = a816007723db93411b98cba454452b37
+                sha1sum_initrd = f0d51189f496e6ad8ed9336bd1128f8a
+        s390x:
+            kernel_params = "console=ttysclp0 serial"
+            boot_path = images
+            kernel = images/f31-s390x/kernel.img
+            # Anaconda hardcodes headless installation of F28 on s390x
+            vga = none
+            inactivity_watcher = none
+            take_regular_screendumps = no
+            unattended_install.cdrom:
+                md5sum_cd1 = 2694a40333d0f698fa9b4f7faea18ad7
+                md5sum_1m_cd1 = 4b6e4bbfa315283235ea014245b21d1d
+            unattended_install.url:
+                sha1sum_vmlinuz = 956a22a1134972dc08422b13595aac53
+                sha1sum_initrd = 735a5698f95463f4fbf7e1c90a75c1a2
+        x86_64:
+            kernel_params = "console=tty0 console=ttyS0"
+            unattended_install.cdrom:
+                md5sum_cd1 = 18740b445159c54d10bd887650e8d1d7
+                md5sum_1m_cd1 = cf06d7246d515d06ebb73e6e4a2f547f
```

Comment:
Just downloaded this one and it is suppose to be:

```
md5sum_cd1 = af29363e2c8825948515aac36701fbfa
md5sum_1m_cd1 = 8c09623eee12715472242f844cf69e4e
```

please download the rest and update it accordingly.

Reply:
If unsure you can use:

```
for AAA in *.iso; do
echo "$AAA"
md5sum "$AAA"
dd if="$AAA" bs=1M count=1 | md5sum
done
```
which should list md5sum and md5sum_1m for each iso in the current folder.

Reply:
Sure,thanks a lot for your review and help me check this.
Btw,should we change sha1sum_vmlinuz(initrd) to md5sum_vmlinuz(initrd),
or update the hashcodes with sha1sum ones,or remain the same?

Reply:
Hi Lukas,
I chose the second way and updated the commit and modified the 28.cfg as the links are outdated,please feel free to tell me if they are not wanted.
Btw, I use head -c 1M xxx | md5sum to create the hashcode on the first 1M  before see your comment.

Reply:
no problem, that is equal ;-)